lib/tasks/generate_lat_long.rake

```diff
@@ -0,0 +1,31 @@
+# lib/tasks/generate_lat_long.rake
+
+namespace :generate_lat_long do
+  desc 'Generate latitude and longitude for all finishers'
+
+  task :update => :environment do
+    finishers = Finisher.all
+
+    # iterate through each finisher and attempt to geocode them...
+    finishers.each do |finisher|
```

Comment:
I'd use batches on this - check out https://api.rubyonrails.org/v7.0.7.2/classes/ActiveRecord/Batches.html for examples.